src/pheval/cli_pheval_utils.py

```diff
@@ -532,3 +485,48 @@ def benchmark_comparison(
         disease_analysis,
         plot_type,
     )
+
+
+@click.command("semsim-to-exomiserdb")
+@click.option(
+    "--input",
+    "-i",
+    required=True,
+    metavar="FILE",
+    help="Semsim input file.",
+    type=Path,
+)
+@click.option(
+    "--object-prefix",
+    required=True,
+    metavar="object-prefix",
+    help="Object Prefix. e.g. MP",
+    type=str,
+)
+@click.option(
+    "--subject-prefix",
+    required=True,
+    metavar="subject-prefix",
+    help="Subject Prefix. e.g. HP",
+    type=str,
+)
+@click.option(
+    "--db-path",
+    "-d",
+    required=True,
+    metavar="db-path",
+    help="Exomiser Phenotypic Database Folder Path. (e.g. /exomiser_folder/2209_phenotype/2209_phenotype/)",
+    type=Path,
+)
+def semsim_to_exomiserdb_command(
+    input: Path, object_prefix: str, subject_prefix: str, db_path: Path
```

Comment:
It would be good to rename `input` to something more specific, as it shadows a Python built-in